Providers/nxOMSAutomationWorker/automationworker/worker/omsutil.py

```diff
@@ -79,6 +79,16 @@ def initialize():
     else:
         print "Successfully set owners of certificate folder."
 
+    proxy_paths = ["/etc/opt/microsoft/omsagent/conf/proxy.conf", "/etc/opt/microsoft/omsagent/proxy.conf"]
+    for path in proxy_paths:
+        if os.path.exists(path):
+            process, output, error = linuxutil.popen_communicate(["sudo", "chmod", "g+r", path])
```

Comment:
I think, if you make this change, you should also make the change in omsadmin.sh to change the proxy.conf perms from 600 to 640.